agent/lib/pbench/add_metalog_option.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+
+# Usage: pbench-add-metalog-option <metadata log file> <section> <option>
+
+# Add an option to a section of the metadata.log file.
+# E.g. using an 'iterations' arg for the option
+#
+# iterations: 1-iter, 2-iter, 3-iter
+#
+# where the iterations are in the <iterations.file>, one iteration per line.
+import os
+import sys
+from configparser import ConfigParser, NoSectionError
+
+
+def add_metalog_option(log_file, section, option, value):
+    if not os.path.exists(log_file):
```

Comment:
Typically, the "pythonic" way to approach this is something like:

```
try:
    config.read(log_file)
except OSError as exc:
    if exc.errno == errno.ENOENT:
        raise Exception(f"Log file does not exist: {log_file}")
    raise
```

This lets us avoid extra tests and code to support those tests.
```